code/game/gamemodes/wizard/wizard.dm

```diff
@@ -9,162 +9,33 @@
 	required_players = 20
 	required_enemies = 1
 	recommended_enemies = 1
-	enemy_minimum_age = 14
+	enemy_minimum_age = 0

```

Comment:
wizard is not that complicated. certanly less complicated then rev or cult which you can end up in day one.